docs/configuration.md

```diff
@@ -10,7 +10,7 @@ Honeycomb uses environment variables during build and run time to control the di
 * `ELECTRON_START_URL` ![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow): URL (e.g. `http://localhost:3000`) where the front end of the app is being hosted - also used in `electron.js` to indicate the app is running in dev mode
 * `EVENT_MARKER_PRODUCT_ID`![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow)![](https://img.shields.io/badge/-runtime-purple): The product ID of the event marker (e.g. `0487`).  If not set, it will use the `productID` set in `public/config/trigger.js` if available, or attempt to connect using the com name.
 * `EVENT_MARKER_COM_NAME` ![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow)![](https://img.shields.io/badge/-runtime-purple): The com name of the event marker (e.g. `COM3`). If not set, it will use the `comName` set in `public/config/trigger.js`.  If the productID is set (not an empty string), this field will be ignored.
-* `REACT_APP_VOLUME`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-runtime-purple)![](https://img.shields.io/badge/-buildtime-blue): whether the participant is being asked to adjust volume. This can be used when the task is running as a desktop app (electron). Not defining this variable is equivalent to setting it to `false`. Build-time and run-time values need to be the same.
+* `REACT_APP_VOLUME`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-buildtime-blue): whether the participant is being asked to adjust volume. This can be used when the task is running as a desktop app (electron). Not defining this variable is equivalent to setting it to `false`. Build-time and run-time values need to be the same.
```

Comment:
We should discuss the last line of these paragraphs with @mcmcgrath13 . But if build-time and run-time values need to match, then we should try not to expose... @suivo2001 maybe we should send them all to the config file as you did [here](https://github.com/brown-ccv/honeycomb/blob/fc9885563d1a7074bfbffe50b063e494dc6635d1/package.json#L51}

Reply:
@suivo2001 did you do the testing on whether these are runtime? I don't believe they are (for the `REACT_APP...` ones.